rust/sealevel/programs/hyperlane-sealevel-igp/src/processor.rs

```diff
@@ -319,6 +315,20 @@ fn pay_for_gas(program_id: &Pubkey, accounts: &[AccountInfo], payment: PayForGas
     // Make sure an account can't be written to that already exists.
     verify_account_uninitialized(gas_payment_account_info)?;
 
+    // Account 5: The IGP account.
```

Comment:
Was this code moved down here to have it right next to the overhead igp one?

Reply:
Yeah exactly. After moving warp routes over to paying for gas I realized that crafting the account metas when using an overhead IGP is more inconvenient if the overhead IGP & inner IGP aren't specified right after one another